examples/azure_file/azure.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+ name: azure
+spec:
+ containers:
+  - image: kubernetes/pause
+    name: azure
+    volumeMounts:
+      - name: azure
+        mountPath: /mnt/azure
+ volumes:
+      - name: azure
+        azureFile:
+          accountName: account_name
+          accountKey: account_key
```

Comment:
azure has separate storage and compute accounts. In AWS and GCE we use instance because we provision block storage to instance. Azure File storage can be provisioned without an instance. You can even access file share off Azure cloud.